src/components/review/StorageReview.jsx

```diff
@@ -86,11 +87,18 @@ const DeviceRow = ({ disk }) => {
         const size = cockpit.format_bytes(devices[device].size.v);
         const request = requests.find(request => request["device-spec"] === device);
         const format = devices[device].formatData.type.v;
-        const action = (
-            request === undefined || request.reformat
-                ? (format ? cockpit.format(_("format as $0"), format) : null)
-                : ((format === "biosboot") ? format : _("mount"))
-        );
+
+        let action = null;
+        if (reuseHomeRequest?.["reused-mount-points"].includes(mount)) {
+            action = _("reuse");
```

Comment:
reuse is essentially just doing the mount. IOW technically it's correct to write 'mount' there. I dont think we should explicitly mention reuse in the review page. Was this like this in the mockups? @garrett

Reply:
Agreed. Good point.

Reply:
Updated.